R/module_data_summary.R

```diff
@@ -141,16 +140,13 @@ srv_data_summary <- function(id, teal_data) {
 get_filter_overview <- function(teal_data) {
   datanames <- teal.data::datanames(teal_data())
   joinkeys <- teal.data::join_keys(teal_data())
+
   filtered_data_objs <- sapply(
     datanames,
     function(name) teal.code::get_env(teal_data())[[name]],
     simplify = FALSE
   )
-  unfiltered_data_objs <- sapply(
-    datanames,
-    function(name) teal.code::get_env(teal_data())[[paste0(name, "._raw_")]],
-    simplify = FALSE
-  )
+  unfiltered_data_objs <- teal.code::get_env(teal_data())[[".raw_data"]]
```

Comment:
if we already using API from `teal.code`, maybe `teal.code::get_var()`?
not a big deal though

Reply:
addressed